packages/zowe-explorer/__tests__/__unit__/utils/ProfilesUtils.unit.test.ts

```diff
@@ -352,6 +346,63 @@ describe("ProfilesUtils unit tests", () => {
             expect(mockReadProfilesFromDisk).toHaveBeenCalledTimes(1);
             profInfoSpy.mockRestore();
         });
+        it("should prompt user if v1 profiles detected and call execute command with Create New chosen", async () => {
+            const mockReadProfilesFromDisk = jest.fn();
+            const profInfoSpy = jest.spyOn(profUtils.ProfilesUtils, "getProfileInfo").mockReturnValueOnce({
+                readProfilesFromDisk: mockReadProfilesFromDisk,
+                usingTeamConfig: false,
+                getTeamConfig: jest.fn().mockReturnValue([]),
+                getAllProfiles: jest.fn().mockReturnValue([createValidIProfile(), createAltTypeIProfile()]),
+            } as never);
+            const msgSpy = jest.spyOn(Gui, "infoMessage").mockResolvedValueOnce("Create New" as any);
+            const commandSpy = jest.spyOn(vscode.commands, "executeCommand");
+            await expect(profUtils.ProfilesUtils.readConfigFromDisk()).resolves.not.toThrow();
+            expect(msgSpy).toHaveBeenCalledWith(
+                "Zowe v1 profiles in use.\n  Zowe Explorer no longer supports v1 profiles, choose to convert existing profiles to a team configuration or create new.",
```

Comment:
Hi @JillieBeanSim, this is looking good. Could you remove the space after the "\n" here?

As it is, the space is noticeable on Windows: 
![image](https://github.com/zowe/vscode-extension-for-zowe/assets/122040687/0c60e209-fc99-48d6-90ff-ede0fa7406cc)